components.api

```diff
@@ -289,7 +289,7 @@ Ubuntu.Components.Popups.ComposerSheet 1.3: SheetBase
 Ubuntu.Layouts.ConditionalLayout 1.0 0.1 ULConditionalLayout: QtObject
     default property Component layout
     property string name
-    property QQmlBinding when
+    property bool when
```

Comment:
`QQmlBinding` was coming from private API. Something had changed in Qt such that the code for the layouts plugin would crash when calling `evaluate()->toBool()` on the binding. Since it expects a bool there anyway, and Qt already internally does appropriate binding, it was better to get rid of the dependency on the private API and just use a bool instead.

Reply:
Ok, so the old code was crashing.